lib/ymlr/encoder_test.exs

```diff
@@ -168,6 +168,18 @@ defmodule Ymlr.EncoderTest do
       assert MUT.to_s!(%TestStruct{foo: 1, bar: 2}) == "bar: 2\nfoo: 1"
     end
 
+    test "keyword lists" do
+      assert MUT.to_s!([a: 1]) == "a: 1"
+      assert MUT.to_s!([a: 1, b: 2]) == "a: 1\nb: 2"
+      assert MUT.to_s!([a: nil]) == "a:"
+    end
+    
+    test "invalid keyword key" do
+      assert_raise ArgumentError, fn ->
+        MUT.to_s!([{"a", 1}])
```

Comment:
don't think I agree with this: it is not a valid keyword list, but it is a valid list with 1 tuple element ...

so, still not supported but the name of the test should probably be "list of tuples - tuples not supported"

=> and we should probably also add a "tuples not supported" test to make it explicit

Reply:
new tests done in e5460045b68399b16d46271bcd6494776850ccfa